docs: improve governance (#9)
yeoman/.github: .github/CODE_OF_CONDUCT.md

# Code of Conduct

"We promise to extend courtesy and respect to everyone involved in this project regardless of gender, gender identity, sexual orientation, disability, age, ethnicity, religion, level of experience, or any other aspect of life. We expect anyone contributing to the Yeoman project to do the same."

All contributions to this project are valuable, including and not limited to posting issues, updating documentation, submitting pull requests and providing feedback in comments. Please act with respect and avoid demeaning, condescending, racist, sexist and other inappropriate language and conduct. 

Please ensure comments stay professional, constructive and about the code, not the person. 

> [![xkcd](xkcd-1053.png)](http://xkcd.com/1053/)

Reviewer: ❤️ love this comment, +1!

Touching up the alt text to make it clear it's this comic:

```suggestion
[![xkcd comic 1053](xkcd-1053.png)](http://xkcd.com/1053/)
```

> This Code of Conduct applies to all of the projects under the [Yeoman organization](https://github.com/yeoman) on GitHub and the Yeoman community at large (IRC, mailing lists, Google+, Twitter, etc.).

Reviewer: > Google+, Twitter

😄 I wonder what year this was first written...

To make it more future-proof:

```suggestion
This Code of Conduct applies to all of the projects under the [Yeoman organization](https://github.com/yeoman) on GitHub and the Yeoman community at large (Discord, mailing lists, social media, etc.).
```

### How to Submit An Issue

To submit an issue, simply click the Issues tab in the far right menu, below the <> Code tab.

On the far right of that screen is a big green button that says "New Issue". Click the button and it will take you to a screen like this:

Please make the Title of your issue "Code of Conduct Content Update"

Issues will be reviewed by the Yeoman core team. This is standard for the GitHub workflow, which works primarily to serve code reviews. Any delay in time between your request and its addition to the Code of Conduct is not an indication it has been received negatively.

Please note that some edits may be made to your suggestion, and this is also standard for the open source community and GitHub workflows. 

We welcome your thoughts and contributions. 

## Consequences for violations to this Code of Conduct

If a participant engages in any behaviour violating this code of conduct, the core members of this community may take any action they deem appropriate, including warning the offender or expulsion from the community, exclusion from any interaction and loss of all rights in this community. 

## Reporting violations of the Code of Conduct

> If you personally require help or you are witness to violations of the Code of Conduct, please immediately contact  [Any Core Team Member](https://github.com/yeoman/yeoman/#team).

Reviewer: [Formatting] Nit:

```suggestion
If you personally require help or you are witness to violations of the Code of Conduct, please immediately contact [Any Core Team Member](https://github.com/yeoman/yeoman/#team).
```
